internal/timecraft/process.go

```diff
@@ -59,10 +59,16 @@ type ProcessInfo struct {
 	// ID is the ID of the process.
 	ID ProcessID
 
+	// Addr is a unique IP address for the process.
+	Addr netip.Addr
+
```

Comment:
This is good enough for now but I think we'll want to enhance the model later to have a representation of each network interface exposed to the process :)